### DISCUSSION TITLE
Which distributions are supported by which installation method?

### DISCUSSION THREAD

**Discussion by CodesInChaos** [geldata/gel#1573] [Q&A] [answered]
There seem to be three installation methods:

1. edgedb-cli native
2. edgedb-cli docker
3. package manager

I tried edgedb-cli native on Mint 18 and 20 and it didn't work, claiming that ubuntu isn't supported (at different times in the installation process).

I tried the package manager on Mint 18 and it failed with a libc version conflict.

I tried the package manager on Mint 20 and it worked to some degree. Since edgedb-cli didn't find the installation, I had to start the server manually and connect to in a different way from what was described in the tutorial.

Which distributions does edgedb-cli support natively? Could you update the download page to mention which distributions edgedb-cli is compatible natively/via docker?

**Comment by elprans** [comment, marked as answer]
We currently build native packages for:

- Ubuntu {16,18,20).04
- Debian {Stretch,Buster}
- RHEL/Centos 7/8

In addition, there's the `edgedb/edgedb` docker image that should work on any x86_64 OS where Docker is a thing.

Installation via `edgedb-cli` is a new feature, we've just landed it, so the scope of support is a moving target (you can see from the list of [issues](https://github.com/edgedb/edgedb-cli/issues)). The setup flow will see lots of improvement in the near term. Also, Mint is Debian-based, and so should be easy to add support for.

**Reply by 1st1** [reply]
I've created an issue https://github.com/edgedb/edgedb-cli/issues/109

**Reply by CodesInChaos** [reply]
So edgedb-cli based installation works on ubuntu? Because the error message I received explicitly mentioned ubuntu being unsupported. In that case I'll just switch to an ubuntu VM.

**Reply by elprans** [reply]
The tool got confused by seeing Ubuntu-like OS, but not a specifically-supported flavor so you got a misleading error message. Will fix.

**Comment by tailhook** [comment]
Update: we support docker in the CLI tool since `1-alpha.6`.
